src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -49,6 +49,12 @@
   - name: Gets health state of the environment, without stopping on first error.
     text: >
         az acr check-health --ignore-errors
+  - name: Verifies if specific blob identified by image 'sha256:abc123' is available to pull from the repository.
```

Comment:
Maybe use artifact instead of blob here?

Reply:
Consulted with Esteban on this one, will be sticking with the word 'blob' as the command is checking if blobs within a given image can be pulled from current environment, but will update description.